stable_baselines3/common/policies.py

```diff
@@ -53,7 +53,7 @@ def forward(self, observations: th.Tensor) -> th.Tensor:
 class NatureCNN(BaseFeaturesExtractor):
     """
     CNN from DQN nature paper: https://arxiv.org/abs/1312.5602
```

Comment:
I am removing this reference in #35, to avoid any confusion as to what is the network architecture.